pkg/onit/setup2/env.go

```diff
@@ -0,0 +1,99 @@
+package setup2
+
+// TestEnv provides the test environment to tests
+type TestEnv interface {
```

Comment:
Where this interface is implemented? Is it supposed to call the functions under setup interface?